docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -258,7 +288,51 @@ The code snippet below demonstrates how to use dynamic batching in simple scenar
          :fragment: dynamic_batch
 
 
-For more details, see the :doc:`dynamic shapes guide<openvino_docs_OV_UG_DynamicShapes>`.
+Notes for performance and memory consumption in dynamic shape
+-----------------------------------------------------------
+
+- Extra CPU utilization during inference :
+
+   - Shape inference for new input shapes
+   - Kernel compilation in runtime for optimal kernel
```

Comment:
Need to mention that it can be controlled by compilation_num_threads and host_task_priority property

Reply:
Actually the async kernel compilation uses fixed # of threads : 4, because we observed that the inference perf was not good if we use same # of threads as compile_model. It is currently not configurable, and would we need to make it configurable in this release?

Reply:
Oh, I missed that nstreams for compilation task executor is overridden.
As for configurability, I think it's not that critical, but we need to consider some knobs for future releases (to control number of threads for compilation, weights and kernels caches capacity, etc)

Reply:
Okay! Lets push out adding new config for this to the next release.